### PR TITLE
Please comment on this project, I NEED HELP

### PR DESCRIPTION
**Pull request by GregCrutchlow**
Code is jank

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.3",
     "react-scripts": "5.0.1",
+    "sass": "^1.55.0",
```
Comment (PPableo) [inline package.json:14]: not being utilized from the looks of it, might be worth doing uninstalling to clean up files.

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
-        </a>
-      </header>
-    </div>
-  );
+		<div className="wrapper">
+			<header>
+				<Link to="/">
+          <img src={logo} alt="magic the gathering logo" />
+          <h1>Card Finder</h1>
+          <hr />
```
Comment (PPableo) [inline src/App.js]: not sure if this is proper indentation when it comes to nesting although I do see how it's easier to read.

File: src/CardInfo.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		axios({
+			url: `https://api.magicthegathering.io/v1/cards/${urlParamsValue}`,
+			
+		}).then((allMagicCards) => {
+			setCardData(allMagicCards.data.card)
+			console.log(allMagicCards.data.card)
+		})
+	},[urlParamsValue])
+
+    const { name, imageUrl, manaCost, colors, type, rarity, setName, text, power, toughness, flavor } = cardData
```
Comment (PPableo) [inline src/CardInfo.js:22]: great destructing!

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
+				</Link>
+			</header>
+			<Routes>
+				<Route path="/" element={<Enter />} />
+				<Route path="/searchcards" element={<SearchCards />} />
+				<Route path="/cards/:id" element={<CardInfo />} />
+			</Routes>
```
Comment (PPableo) [inline src/App.js:21]: Good use of Routes/Route we learned!

File: src/App.css [hunk 5]
```diff
@@ -36,3 +36,171 @@
...
+  }
+}
+
+@media (max-width: 460px) {
+    header img {
+    display: block;
+    max-width: 80%;
+    height: auto;
+  }
+}
```
Comment (PPableo) [inline src/App.css:233]: Great use of media queries fully tested and working!

File: src/SearchCards.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+		})
+	}, [])
+
+	return (
+		<ul className="catalogue">
+			{magicCards.map((cardsObj) => {
+				return (
+					<li key={cardsObj.id}>
+						<Link to={`/cards/${cardsObj.id}`}>
+							<img src={cardsObj.imageUrl} alt={cardsObj.name} />
```
Comment (kwametsunami) [inline src/SearchCards.js:31]: you can setup an onError catch to have a placeholder image if a card doesn't load or if there's issues!

File: src/CardInfo.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+    return (
+        <div className="cardContainer">
+            <div className="cardInfoContainer">
+                <h2>{name}</h2>
+                <p>Type: {type}</p>
```
Comment (kwametsunami) [inline src/CardInfo.js]: you can try to have span's on the type, cmc, colours, etc so you can style them differently (bold, or change size) from the received data

File: src/CardInfo.js (hunk 3 above)
Comment (kwametsunami) [reply src/CardInfo.js:22]: agreed!

File: src/App.css (hunk 5 above)
Comment (kwametsunami) [inline src/App.css:233]: awesome css, very organized!

File: src/App.css [hunk 8]
```diff
@@ -14,8 +14,8 @@
 }
 
 .App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+  background-color: white;
```
Comment (kwametsunami) [inline src/App.css]: i'm stealing adrian's comment and saying to keep colors to either names, hex codes, or rgb lol

**Review by kwametsunami — COMMENTED** [review]
great syntax, logic, and site -- code looks great!

File: package.json (hunk 1 above)
Comment (GregCrutchlow) [reply package.json:14]: Deleted from app. Thanks!

File: src/CardInfo.js (hunk 3 above)
Comment (GregCrutchlow) [reply src/CardInfo.js:22]: Thanks!

File: src/App.css (hunk 5 above)
Comment (GregCrutchlow) [reply src/App.css:233]: Thank you thank you.

File: src/SearchCards.js (hunk 6 above)
Comment (GregCrutchlow) [reply src/SearchCards.js:31]: Thats a great suggestion and was something I was playing with but came up with a different work around. Thanks!

File: src/App.css (hunk 8 above)
Comment (GregCrutchlow) [reply src/App.css]: I only do everything.

**Pull request merged**